src/main/java/com/exasol/adapter/dynamodb/mapping/ToJsonPropertyToColumnMapping.java

```diff
@@ -11,8 +11,8 @@
  * Maps a property of a DynamoDB table and all its descendants to a JSON string.
  */
 public final class ToJsonPropertyToColumnMapping extends AbstractPropertyToColumnMapping {
-    private static final long serialVersionUID = -5414384284778565811L;
-    private final int maxLength;
+    private static final long serialVersionUID = -836984641709778657L;
+    private final int exasolStringSize;
```

Comment:
What does exasol string mean? Maybe it's better to rename this variable somehow

Reply:
Does it mean varchar?

Reply:
maybe maxSize is clearer?